.npmignore

```diff
@@ -0,0 +1,2 @@
+.github
+
```

Comment:
Since we are creating a `.npmignore` here it would be best to ignore other items such as:

```ignore
test/
misc/
.travis.yml
Makefile
```

Reply:
True. I didn't realize until double checking the docs on `.npmignore` that NPM uses `.gitignore` if you have no `.npmignore`. So we need to add this stuff back in so that we continue to get the behavior we've had so far. Changed.